packages/site/src/ui/index.js

```diff
@@ -1,25 +1,49 @@
-export ChangeLog from './change-log'
```

Comment:
should we config babel to allow this type of export?

Reply:
I chose not to just for the sake of writing more "javascript today"-style code.  It also meant less configuration.